README.md

```diff
@@ -20,7 +20,9 @@ Please see [CONTRIBUTING](CONTRIBUTING.md) for details.
 
 ## Community Projects
 
-- **and more...**
+- [lecodingdev/rwid-git](https://github.com/lecodingdev/rwid-git)
+
+* **and more...**
```

Comment:
"and more section" is also a unordered list. Please don't remove the "-" mark.

commit message: `style: fix removed unordered list at "and more"`

Reply:
done add "-" mark to make "and more section" in unordered list.

Reply:
Please add this line above your lecoding/dev ... repository to prevent merge conflict.

- [ianriizky/rwid-git](https://github.com/ianriizky/rwid-git)